performanceMetrics/src/App/App.js

```diff
@@ -17,6 +17,7 @@ import Chart from '../views/Chart';
 import css from './App.module.less';
 
 const listOfComponents = [
+	'Overall',
```

Comment:
If Overall.txt exists from the new version, a console error occurs in the previous version.
There may be differences in version, so please handle this case so that there is no console error.

I tested with 2.0.0 and 2.0.1 folder and that overall.txt only exists in 2.0.1.
![image](https://user-images.githubusercontent.com/34676178/153536847-bacbe6b2-2f17-473d-aa6d-c394d3881d4f.png)

Reply:
I meant you can hide console errors.
https://stackoverflow.com/questions/48298890/axios-how-to-get-error-response-even-when-api-return-404-error-in-try-catch-fi

![image](https://user-images.githubusercontent.com/34676178/153781318-373d3d5c-a2d5-48c5-8439-3b3eb188a1c1.png)
When I open enact01 index js, 98 errors occur.

As I tested. "axios.get(url, { validateStatus: false })" works even if  you use `Promise.allSettled`

Reply:
Unfortunately, this cannot be solved programatically. The message is printed by the browser, not the app.
https://stackoverflow.com/a/42986081/6050306

Reply:
OK... :(